tests/test_parser.py

```diff
@@ -31,28 +41,103 @@ def bytes_to_service_info(payload: bytes) -> BluetoothServiceInfo:
     )
 
 
+def bytes_to_service_info_wrong_mfr(payload: bytes) -> BluetoothServiceInfo:
+    return BluetoothServiceInfo(
+        name="Test",
+        address="AB:CD:EF:BA:DC:FE",
+        rssi=-60,
+        manufacturer_data={1234: payload},
+        service_data={},
+        service_uuids=[],
+        source="",
+    )
+
+
 def test_parsing_v5():
     device = RuuvitagBluetoothDeviceData()
-    advertisement = bytes_to_service_info(V5_OUTDOOR_SENSOR_DATA)
+    advertisement = bytes_to_service_info(V5_SENSOR_DATA)
     assert device.supported(advertisement)
     up = device.update(advertisement)
-    expected_name = "RuuviTag EFAF"
+    expected_name = "RuuviTag DCFE"
```

Comment:
These seem like spurious changes too..?

Reply:
I guess as mentioned in the previous MR I wasn't really sure why "OUTDOOR" needed to be specified.  The parser doesn't care if data is outdoor or indoor, the unit tests only need verify that the outputs match the inputs.  I added a more "diverse" input in the V3 test but it'd be easy for me to generate a few more based on the tags I have if you think that's necessary (I have some in a freezer, some in a hot attic, etc).  I think we're pretty well covered at this point though.  The MAC change is because the original test had a mac of `00:00:00:00:00:00` and I didn't think that was a very good test.